docs/www/quick-start-kubernetes.md

```diff
@@ -14,8 +14,14 @@ To get up and running you need to create a cluster and deploy the Redpanda opera
 - For production or benchmarking, set up a [production deployment](/docs/production-deployment).
 - You can also set up a [Kubernetes cluster with external access](/docs/kubernetes-external-connect).
 
-> **_Note_** - Run a container inside the Kubernetes cluster to communicate with the Redpanda cluster.
-> Currently, a load balancer is not automatically created during deployment by default.
+**_Note_** - Run a container inside the Kubernetes cluster to communicate with the Redpanda cluster.
+Currently, a load balancer is not automatically created during deployment by default.
+
+**_Note_** - In the steps below, the `.yaml` file that you use to install Redpanda sets `developerMode: true`. 
+If you you want to set `developerMode: false` after the installation, for optimal configuration we recommend that you run `rpk redpanda tune all` directly on the host before you create a Redpanda cluster.
```

Comment:
We need to remove "after the installation" (that change in the yaml would typically happen prior to the installation). 

The rest is just right.

Reply:
Got it! https://deploy-preview-452--vectorizedio.netlify.app/docs/quick-start-kubernetes

Looks good?

Reply:
just noticed this typo "recommentations"

Reply:
One more time!  https://deploy-preview-452--vectorizedio.netlify.app/docs/quick-start-kubernetes

Reply:
@dimitriscruz - one last question - do we want to explain why we're setting fs.aio-max-nr to 1048576 or greater?